### PR TITLE
Change social dominance old API

### PR DESCRIPTION
**Pull request by spiderjako**
## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have tried to find clearer solution before commenting hard-to-understand parts of code
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: lib/sanbase_web/graphql/resolvers/social_data_resolver.ex [hunk 1]
```diff
@@ -155,10 +156,26 @@ defmodule SanbaseWeb.Graphql.Resolvers.SocialDataResolver do
 
   def social_dominance(
         _root,
-        %{slug: slug, from: from, to: to, interval: interval, source: source},
+        %{slug: _slug, from: _from, to: _to, interval: _interval, source: source} = args,
         _resolution
       ) do
-    SocialData.social_dominance(%{slug: slug}, from, to, interval, source)
+    source =
```
Comment (IvanIvanoff) [inline lib/sanbase_web/graphql/resolvers/social_data_resolver.ex:162]:
```suggestion
    source = if source == :all, do: :total, else: :source
```
you can do it on 1 row also

Comment (spiderjako) [reply lib/sanbase_web/graphql/resolvers/social_data_resolver.ex:162]: ``mix format`` makes it 5 lines of code, wouldn't that be a problem if it were to change?

Comment (IvanIvanoff) [reply lib/sanbase_web/graphql/resolvers/social_data_resolver.ex:162]: `mix format` cannot change between the 2 formats - the one with do..end, and the one with a kw-list (`,do: ... , else: ...`)

Comment (spiderjako) [reply lib/sanbase_web/graphql/resolvers/social_data_resolver.ex:162]: ah, didn't see that it was a different format, thanks

**Review by IvanIvanoff — APPROVED** [review]

**Pull request merged**